doc/index.rst

```diff
@@ -21,7 +21,7 @@ Here are some of Sphinx's major features:
   <builtin-extensions>`, and much more functionality via :ref:`third-party
   extensions <third-party-extensions>`.
 * **Themes:** modify the look and feel of outputs via :doc:`creating themes
-  <development/theming>`, and re-use many :ref:`third-party themes
+  <development/theming>`, and reuse many :ref:`third-party themes
```

Comment:
`re-use` is fine

Reply:
It depends on the definition of fine. The rationale follows:
  * All majors dictionaries suggest `reuse` only:
    * [OED](https://www.oed.com/search/dictionary/?q=re-use)
    * [Cambridge](https://www.oxfordlearnersdictionaries.com/definition/english/reuse_1)
    * [Collins](https://www.collinsdictionary.com/dictionary/english/reuse)
    * [Longman](https://www.ldoceonline.com/dictionary/reuse)
    * [Merriam-Webster](https://www.merriam-webster.com/dictionary/reuse)
    * [Oxford Learner's Dictionary](https://dictionary.cambridge.org/dictionary/english/reuse)
  * From [The Blue Book of Grammar and Punctuation](https://www.grammarbook.com/blog/hyphens/hyphens-with-the-prefix-re/):
    > Use the hyphen with the prefix _re_ only when _re_ means _again_ AND omitting the hyphen would cause confusion with another word.
* Both `re-use` and `reuse` can be seen as correct, but:
  * The [Google Ngram Viewer](https://books.google.com/ngrams/graph?content=re-use%2Creuse&year_start=1800&year_end=2019&corpus=en-2019) shows `reuse`  wider use (not by much I must admit).

I do understand you have your own preference, but for a documentation that targets a large audience, what's wrong with following dictionaries, grammar rules and common use?

Reply:
Thanks for the explanation, I'm content to adopt your reccomendation.